packages/browser-wallet/src/shared/utils/token-helpers.ts

```diff
@@ -77,26 +77,32 @@ function deserializeTokenMetadataReturnValue(returnValue: string): string[] {
     return urls;
 }
 
+export enum CIS2ConfirmationError {
+    Cis0Error,
+    Cis2Error,
+}
+
 /**
  * Confirms that the given smart contract instance is CIS-2 compliant
  */
 export async function confirmCIS2Contract(
     client: JsonRpcClient,
     { contractName, index, subindex }: ContractDetails
-): Promise<string | undefined> {
+): Promise<CIS2ConfirmationError | undefined> {
     const supports = await client.invokeContract({
         contract: { index, subindex },
         method: `${contractName}.supports`,
         parameter: getCIS2Identifier(),
     });
     if (!supports || supports.tag === 'failure') {
-        return 'Chosen contract does not support CIS-0';
+        return CIS2ConfirmationError.Cis0Error;
```

Comment:
Why not just use the i18n directly instead of creating this enum?

Reply:
Because this is a "helper" file without access to translations. Unless theres a way to do that, which I don't know of?